tests/utilities/nifgen_measurement/NIFgenMeasurement.serviceconfig

```diff
@@ -0,0 +1,19 @@
+{
+  "services": [
+    {
+      "displayName": "NI-Fgen Measurement (Py)",
```

Comment:
The correct capitalization is "NI-FGEN": https://www.ni.com/en/support/downloads/drivers/download.ni-fgen.html#494664

This applies to English text like display names and descriptions. It doesn't apply to programmatic ids like service classes; keep using PascalCase for those.

Reply:
Sure. Thank you, I'll keep these distinctions in mind.